Neos.ContentRepository.Core/Classes/NodeType/NodeType.php

```diff
@@ -490,11 +493,34 @@ public function hasAutoCreatedChildNode(NodeName $nodeName): bool
      */
     public function getTypeOfAutoCreatedChildNode(NodeName $nodeName): ?NodeType
     {
+        $this->initialize();
```

Comment:
good catch!

Reply:
haha actually not the only missing 😂 
https://github.com/neos/neos-development-collection/issues/4333

will revert this and must be fixed correctly in another pr